packages/app/cypress/e2e/settings.cy.ts

```diff
@@ -155,15 +160,28 @@ describe('App: Settings', () => {
       })
     })
 
-    // TODO: The Edit button isn't hooked up to do anything when it should trigger the openFileInIDE mutation (https://cypress-io.atlassian.net/browse/UNIFY-1164)
-    it.skip('opens cypress.config.js file after clicking "Edit" button', () => {
+    it('opens cypress.config.js file after clicking "Edit" button', () => {
+      cy.startAppServer('e2e')
+      cy.withCtx((ctx, o) => {
+        ctx.coreData.localSettings.preferences.preferredEditorBinary = 'computer'
+        o.sinon.stub(ctx.actions.file, 'openFile')
+      })
+
+      cy.visitApp('/settings')
+      cy.findByText('Project Settings').click()
+      cy.findByRole('button', { name: 'Edit' }).click()
+      cy.withRetryableCtx((ctx) => {
+        expect((ctx.actions.file.openFile as SinonStub).lastCall.args[0]).to.eq(ctx.lifecycleManager.configFilePath)
+      })
     })
   })
 
   describe('external editor', () => {
     beforeEach(() => {
       cy.startAppServer('e2e')
-      cy.withCtx(async (ctx) => {
+      cy.withCtx(async (ctx, o) => {
```

Comment:
Is there something that requires this to be async?

Reply:
Nope, removed.